### ISSUE TITLE
Access to the raw message text

### ISSUE THREAD

**Issue by myme** [flint-bot/flint#8]
Using the REST api or ciscospark SDK I'm able to get the raw message text. However, in ~~`bot.hears()`~~ `flint.hears()` it seems like whitespace has been stripped down to a single space? I haven't delved very deep into this yet, but I have a use case where the original whitespace (specifically line breaks) in the message is significant.

**Comment by nmarus** [comment]
You are correct. Here is portion of the code that does that. 

https://github.com/nmarus/flint/blob/master/lib/flint.js#L628

I would be interested in the use case you are running across to better understand. Feel free to drop me an email at nmarus@gmail.com if you are open to share.

**Comment by myme** [comment]
I'm simply creating a bot to create polls for rooms. I want to keep the formatting dead simple and minimize the margin of error, so the title/question is the first line with the subsequent non-empty lines are alternatives. I've already implemented this using `ciscospark`, before I learned about `flint` and decided to port it over. I don't really see a way of getting the same behavior without fetching the message object again (which seems a bit unnecessary.

Example:

```
@Bot /create What's the answer?
1
42
1 million
```
